omod-common/src/main/java/org/openmrs/module/webservices/rest/web/api/impl/RestServiceImpl.java

```diff
@@ -48,8 +48,8 @@
 public class RestServiceImpl implements RestService {
 	
 	volatile Map<String, ResourceDefinition> resourceDefinitionsByNames;
-	
-	volatile Map<Class<?>, Resource> resourcesBySupportedClasses;
+
+	static volatile Map<Class<?>, Resource> resourcesBySupportedClasses;
```

Comment:
Originally, this was not static. Is this just a typo?

Reply:
Did you see the above comment?

Reply:
fixed!